OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop.
+- `PX` - VictorBjelkholm - One or more js-ipfs daemons are part of the IPFS infrastructure (Bootstraper nodes).
+- `PX` - VictorBjelkholm - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon.
+
+### The core can handle vast amounts of data
+
+- `PX` - achingbrain - The MFS is implemented in js-ipfs.
+- `PX` - achingbrain - npm on IPFS is used as a test vector of js-ipfs capabilities.
+- `PX` - OWNER - js-ipfs in the browser can fetch a dataset of at least 100Gb (and create archives.js.ipfs.io).
+- `PX` - OWNER - js-ipfs in Node.js can add and transfer a dataset of at least 1TB.
+- `PX` - jacobheun - A S3 IPFS Repo backend exists and it is easily pluggable on `new IPFS`.
+- `PX` - OWNER - The PIN API is finished and merged.
```

Comment:
Needs an Owner.

Reply:
@JonKrone would you like to be the owner for this one given that you already did most of the work?

Reply:
@diasdavid Yep yep, happy to. I'm sorry for the late response, I misunderstood OKRs.